README.md

```diff
@@ -136,4 +151,17 @@ While the dotnet core cli provides some nuget commands, the process start to fin
 * Reload the changes `source ~/.bash_profile && source ~/.bashrc`
 * Now you can...
     * Run `and-cli` in the root of any repo with the `and-cli` npm package installed
-    * Run `and-cli-dev` in parallel with any active stable versions installed globally with npm
\ No newline at end of file
+    * Run `and-cli-dev` in parallel with any active stable versions installed globally with npm
+
+---
+
+## Troubleshooting
+
+### Value for command arguments are out of order
+Depending upon the shell/terminal you are using, the node process sometimes requires the `--` delimiter between the command and the arguments. Otherwise, especially in a shell like windows command prompt, the value for arguments gets piped out of order.
+
+Example:
+* Before: `and-cli dotnet --cli "test db migrate"`
+    * Works in most shells, but requires the arguments to be in quotes. Fails in windows command prompt
+* After: `and-cli dotnet -- --cli test db migrate`
```

Comment:
I think this answers my above question

Reply:
Yeah, this is intentional and applies to all commands. Just so happens this is the first to have the multiple values to an option